lib/src/main/java/uk/gov/di/ipv/cri/drivingpermit/library/domain/CheckDetails.java

```diff
@@ -3,7 +3,9 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import uk.gov.di.ipv.cri.common.library.annotations.ExcludeFromGeneratedCoverageReport;
 
+@ExcludeFromGeneratedCoverageReport
```

Comment:
Why are we excluding these models from the coverage report?

Reply:
see DocumentCheckResultItem, but the addition of only dl has check details implemented with a dedicated bean rather than just a list of check/fail strings.